receiver/otlpreceiver/config.go

```diff
@@ -83,6 +85,14 @@ func (cfg *Config) Unmarshal(conf *confmap.Conf) error {
 		if cfg.HTTP.LogsURLPath, err = sanitizeURLPath(cfg.HTTP.LogsURLPath); err != nil {
 			return err
 		}
+		//set default  server config value
+		if cfg.HTTP.ReadTimeout == 0 {
+			cfg.HTTP.ReadTimeout = defaultTimeout
+		}
+
+		if cfg.HTTP.WriteTimeout == 0 {
+			cfg.HTTP.WriteTimeout = defaultTimeout
```

Comment:
at the very least this should be spelled out on the README.

This is not the right place to add this default value - it should be done in `createDefaultConfig` instead:
https://github.com/open-telemetry/opentelemetry-collector/blob/2ccef9d0c51b5636f745746b268b598ee2283602/receiver/otlpreceiver/factory.go#L41

We can also change the way the default config is computed to use the confighttp.NewServerConfig and configgrpc.NewServerConfig instead of initializing structs by itself.